src/components/reusable/buttons.tsx

```diff
@@ -0,0 +1,22 @@
+// import button style from ./button.css
+interface ButtonProps {
```

Comment:
Put this inside a separate file called `types.ts` created inside the root folder under **src** folder

Reply:
done